### DISCUSSION TITLE
Stationary Lidar

### DISCUSSION THREAD

**Discussion by tsugg** [microsoft/AirSim#3778] [General]
Hi, I'm experimenting with AirSim to simulate lidar data collection in large buildings. I was hoping to place stationary lidar sensors a couple meters in the air above the areas of interest. I have been able to gather lidar form the car mode, but I was wondering if it was possible to set up a couple unreal cameras and have that as the source of the lidar collection. Is this something that can be done in Computer Vision mode?

**Comment by jonyMarino** [comment]
This can be done in Multirotor mode with the "ExternalPhysicsEngine"
